server/api/cycleData/index.ts

```diff
@@ -23,6 +31,19 @@ export const CycleDataApi = {
     express.get(ApiEndPoint.CycleData.descriptions(), AuthMiddleware.requireView, getDescription)
     express.put(ApiEndPoint.CycleData.descriptions(), AuthMiddleware.requireEdit, upsertDescription)
 
+    // OriginalDataPoints
+    express.get(ApiEndPoint.CycleData.OriginalDataPoint.reservedYears(), AuthMiddleware.requireView, getReservedYears)
+
+    express.post(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, createOriginalDataPoint)
+    express.delete(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, deleteOriginalDataPoint)
+    express.get(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireView, getOriginalDataPoint)
+    express.put(ApiEndPoint.CycleData.OriginalDataPoint.one(), AuthMiddleware.requireEdit, updateOriginalDataPoint)
+
+    express.get(ApiEndPoint.CycleData.OriginalDataPoint.many(), AuthMiddleware.requireView, getOriginalDataPoints)
+
+    // OriginalDataPoint // table
+    express.get(ApiEndPoint.CycleData.OriginalDataPoint.data(), AuthMiddleware.requireView, getOriginalDataPointData)
+
```

Comment:
should we add index.ts with `export OriginalDataPointApi (express) => ..` and replace this with `OriginalDataPointApi.init(express)` ?

Reply:
I think that this way is fine